README.md

```diff
@@ -19,6 +19,28 @@ The outputs are used in
 [PyPSA-Eur](https://github.com/PyPSA/pypsa-eur) and
 [PyPSA-Eur-Sec](https://github.com/PyPSA/pypsa-eur-sec).
 
+## Contributions welcome!
+
+We are happy to receive contributions, either as feedback, corrections, modifications or addition of new technology entries.
+You can either open an *Issue* on GitHub for reporting potentials problems, providing feedback or suggesting additional entries.
+Alternatively you can also directly open a *Pull Request* already containing suggested modifications.
+
+All sizes and qualities of contributions are welcome. After an the *Issue* or *Pull Request* we'll assist with comments and leads to get your contribution included into the repository.
+
+## Unit conventions
+
+The following conventions are recommended for new additions to the repository.
+Care must be taken with legacy entries, of which not all follow these conventions.
+
+* Energy units: Use MWh and MW or kWh and kW
+  Thermal energy content: When referring to the thermal energy content of a mass or volume, use the Lower Heating Value (LHV)
+* Currency-values are specified in EUR
+* Ambigiuous units are avoided by specifying whether the unit applies to the input or the output of a process, e.g. the capacity of hydrogen electrolysis "MW_e" refers to "MW" of electricity input capacity whilst "MW_H2" refers to "MW" of output hydrogen
```

Comment:
The convention seems that capacity is mainly expressed in the output capacity. When there are multiple outputs, the one with the highest exergy is used. Exceptions are electrolysis and solar thermal.

Reply:
* Yes, for some processes like electrolysis and solar thermal it makes sense to specify them based on their input, increasing also the comparability between different electrolysis technologies (and in the future potentially allowing for load-dependent efficiencies).

* I'd say it is less about the exergy content and more about the purpose of the process, i.e. what's the desired output. It should usually overlap, with the "output" definition being easier understood and better applicable to processes like desalination or DAC.